package/Cargo.toml

```diff
@@ -10,10 +10,6 @@ crossbeam = "0.8"
 flate2 = "1.0.22"
 omicron-common = { path = "../common" }
 omicron-zone-package = { version = "0.1.2" }
-# We depend on the propolis-server here -- a binary, not a library -- to
-# make it visible to the packaging tool, which can compile it and shove
-# it in a tarball.
-propolis-server = { git = "https://github.com/oxidecomputer/propolis", rev = "0e3798510ae190131f63b9df767ec01b2beacf91" }
```

Comment:
`sled-agent/Cargo.toml` still points to the corresponding `propolis-client` dep. How do we ensure we're using the right `propolis-server` in this new external package world? Is it just manual at this point?

Reply:
Yeah, `sled-agent` still has that dependency because it's calling into Propolis' API.

Fortunately, for those two components, I expect they'll eventually both be part of the ramdisk - so the versions should be compatible, because they're always bundled together.

However, as the construction of zone images is currently manual, it is theoretically possible they can be built disjointly.

So TL;DR:
- The out-of-sync problem exists,
- but it shouldn't exist when we bundle the zone images into the ramdisk

Reply:
Cool, thanks for clarifying.